src/main/java/org/spdx/storage/listedlicense/LicenseCreatorAgent.java

```diff
@@ -51,6 +52,7 @@ public class LicenseCreatorAgent {
 	private final TypedValue creationInfoTV;
 	
 	public LicenseCreatorAgent(String licenseListVersion) throws SpdxInvalidIdException, SpdxInvalidTypeException, ModelRegistryException {
+		SpdxModelFactory.init();
```

Comment:
```suggestion
               // Call init to make sure the model is initialized before it is used by the license factory methods
               // Note that multiple calls to init does not cause any harm
		SpdxModelFactory.init();
```

Reply:
Not sure if this is the clearest comment, but I wanted to highlight init one should not rely exclusively on this call to init

Reply:
It seems fine.

I would also update the README with something like if you are using the listed license model store, the SPDX model factory will be initialized for you, otherwise you will need to call `SpdxModelFactory.init()` manually.

Reply:
You can push the changes to this PR yourself if you want.